cli/src/main.rs

```diff
@@ -48,15 +48,15 @@ struct Args {
 enum Commands {
     /// Subscribe to mempool accounts
     MempoolAccounts {
```

Comment:
these subcommands are confusing, as they read more like params

maybe add verb in name, e.g. `...  SubscribeMempoolAccounts --acounts ...`

Reply:
updated the readme to be more helpful. will defer to @buffalu